assets/romfs/i18n/de/gui.json

```diff
@@ -0,0 +1,397 @@
+{
+    "LOADER_INSTRUCTIONS_TOP_PRESENT": "Wähle einen Speicherstand zum öffnen. Drücke Y für vorhandene Spiele.",
+    "LOADER_INSTRUCTIONS_TOP_ABSENT": "Wähle einen Speicherstand zum öffnen. Drücke Y für nicht vorhandene Spiele.",
+    "LOADER_INSTRUCTIONS_BOTTOM": "Bewege dein Steuerkreuz. Drücke A zum fortfahren, START zum schließen.",
+    "LOADER_GAME_CARD": "Karte",
+    "LOADER_INSTALLED_GAMES": "Installierte Spiele",
+    "LOADER_ID": "ID: ",
+    "LOADER_MEDIA_TYPE": "Medientyp: ",
+    "LOADER_CARTRIDGE": "Karte",
+    "LOADER_SD": "SD",
+    "LOADER_GAME_SAVE": "Speicherstand",
+    "LOADER_LOAD": "Lade \ue000",
+    "LOADER_WIRELESS": "Kabellos \ue002",
+    "NONE": "Nichts",
+    "OT_NAME": "OT Name",
+    "LANGUAGE": "Sprache",
+    "DEFAULTS": "Standards",
+    "MISC": "Verschiedens",
+    "TID": "TID",
+    "SID": "SID",
+    "OT": "OT",
+    "NATIONALITY": "Nationalität",
+    "DAY": "Tag",
+    "MONTH": "Monat",
+    "YEAR": "Jahr",
+    "CONFIG_BACKUP_SAVE": "Automatisches Backup beim Laden",
+    "CONFIG_STORAGE_SIZE": "Lagergröße",
+    "CONFIG_BAD_SECTORS": "Korrupte Sektoren beim Verlassen reparieren",
+    "CONFIG_EDIT_TRANSFERS": "Bearbeitung während Übertragungen",
+    "CONFIG_BACKUP_INJECTION": "Aktiviere Backup Injektion",
+    "YES": "Ja",
+    "NO": "Nein",
+    "EDITOR_STATS": "STATUSWERTE",
+    "EDITOR_MOVES": "ATTACKEN",
+    "EDITOR_SAVE": "SPEICHERN",
+    "LEVEL": "Level",
+    "ABILITY": "Fähigkeit",
+    "NATURE": "Wesen",
+    "ITEM": "Item",
+    "SHINY": "Schillernd",
+    "POKERUS": "Pok\u00e9rus",
+    "NICKNAME": "Spitzname",
+    "FRIENDSHIP": "Freundschaft",
+    "EDITOR_CP": "CP: ",
+    "IV": "DV",
+    "AWAKENED": "Erwacht",
+    "EV": "EV",
+    "TOTAL": "INSGESAMT",
+    "HP": "KP",
+    "ATTACK": "Angriff",
+    "DEFENSE": "Verteidigung",
+    "SPATK": "Sp.-Ang.",
+    "SPDEF": "Sp.-Vert.",
+    "SPEED": "Initiative",
+    "EDITOR_HIDDEN_POWER": "Kraftreserve ",
+    "MOVES": "Attacken",
+    "RELEARN_MOVES": "Verlernte Attacken",
+    "EDITOR_NOT_APPLICABLE_GEN": "Nicht anwendbar für diese Generation",
+    "EDITOR_INST": "Drücke \ue000 zum auswählen, \ue001 um zurück zu gehen",
+    "BOX_NAME": "Box Name",
```

Comment:
Seems you didn't translate this line.

Reply:
Box and name are the same in German, the part I changed was capitalizing the "name".